iterable-extension/src/main/java/com/mparticle/ext/iterable/IterableExtension.java

```diff
@@ -198,13 +207,7 @@ void updateUser(EventProcessingRequest request) throws IOException {
                 //this is safe due to the filters above
                 updateEmailRequest.newEmail = changeEvent.getAdded().get(0).getValue();
                 Response<IterableApiResponse> response = iterableService.updateEmail(getApiKey(request), updateEmailRequest).execute();
-                // TODO: Decide how to handle
-                if (response.isSuccessful()) {
-                    IterableApiResponse apiResponse = response.body();
-                    if (apiResponse != null && !apiResponse.isSuccess()) {
-                        throw new IOException("Error while calling updateEmail() on iterable: HTTP " + apiResponse.code);
-                    }
-                }
+                handleIterableResponse(response, changeEvent.getId());
```

Comment:
If we're unable to update the user's email, should we stop processing the entire batch?

Reply:
I think it is fine to continue processing. New batches will be referencing the new email anyway, so there isn't any difference if we drop the current batch after failure.